### PR TITLE
Fix/trn 123/externalise mime types

### PR DESCRIPTION
**Pull request by oatymart**
Related to: https://oat-sa.atlassian.net/browse/TRN-123

Depends on: oat-sa/tao-core-sdk-fe#97

The module `qtiCommonRenderer/helpers/data/mimeTypes.js` was moved to `tao-core-sdk` so it can be used by both old and new test runners without creating a dependency between them.

**To test:**
`npm run test`

### CODE REVIEW

**Review by olga-kulish — APPROVED** [review]
Test succeeds, changes look good. Other than that I can't test it.
We shouldn't forget to update `package.json` before merging

**Review by krampstudio — CHANGES_REQUESTED** [review]
I've been able to successfully test it in Terre Deliver
Just a few cosmetic changes before approval

File: test/qtiCommonRenderer/helpers/uploadMime/test.js [hunk 1]
```diff
@@ -0,0 +1,94 @@
+define([
```
[the remark this answers is not in the document]
Comment (oatymart) [reply test/qtiCommonRenderer/helpers/uploadMime/test.js:1]: I can but I'm only following the established pattern in this repo, where no test file has a licence header.

File: test/qtiCommonRenderer/helpers/uploadMime/test.js [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    'taoQtiItem/qtiCommonRenderer/helpers/uploadMime',
+    'taoQtiItem/qtiItem/core/Element'
+], function (
+    uploadMime,
+    Element
+) {
+    'use strict';
+
+    QUnit.module('taoQtiItem/qtiCommonRenderer/helpers/uploadMime', {
+    });
```
[the remark this answers is not in the document]
Comment (oatymart) [reply test/qtiCommonRenderer/helpers/uploadMime/test.js]: Fixed.

File: src/qtiCommonRenderer/helpers/uploadMime.js [hunk 3]
```diff
@@ -13,20 +13,20 @@
...
-     * @TODO these mime types are not up-to-date, in particular the MS ones
-     * refer to http://filext.com/faq/office_mime_types.php
-     * @type [{getMimeTypes: getMimeTypes}]
+     * Fetch mime type definitions
+     * (translation function for labels must be injected from here)
+     * @returns {Array}
      */
     getMimeTypes: function getMimeTypes() {
-        return mimeTypes.getList();
+        return mimeTypes.getList(__);
```
Comment (oatymart) [inline src/qtiCommonRenderer/helpers/uploadMime.js:29]: Note: This is how I adapted this to the signature change in https://github.com/oat-sa/tao-core-sdk-fe/pull/97 (externally-provided translation function).

**Review by krampstudio — APPROVED** [review]
changes looks good

**Pull request merged**